crates/server/README.md

```diff
@@ -0,0 +1,257 @@
+# Calimero Server
+
+- [Introduction](#introduction)
+    - [Admin API](#1admin-api)
+    - [JSON rpc](#2json-rpc)
+    - [Websocket](#3websocket)
+- [Calimero Server Workflows](#calimero-server-workflows)
+    - [Admin API Workflow](#workflow-for-admin-dashboard)
+    - [JSON rpc Workflow](#workflow-for-p2p-web-applications-and-json-rpc)
+    - [Websocket Workflow](#workflow-for-p2p-web-application-and-websocket)
+- [Admin API endpoints](#admin-api-endpoints)
+    - [Protected Routes](#protected-routes)
+    - [Unprotected Routes](#unprotected-routes)
+- [JSON rpc endpoints](#json-rpc-endpoints)
+- [Websocket endpoints](#websocket-endpoints)
+- [Examples](#examples)
+
+## Introduction
+
+The Calimero Server is a server component of the Calimero Node that has crucial part in
+managing the Calimero Admin Dashboard and the P2P Applications built for Calimero Nodes.
+
+Calimero Server component is split into 3 parts:
+
+### 1. Admin API
+
+The Admin API component of the Calimero Server connects web applications with the node and its functionalities. It is primarily utilized by the Admin Dashboard to query and manage various aspects of the node, including:
```

Comment:
Doesn't need to be a web application. It exposes API which can be consumed

Reply:
Updated